frontend/src/components/LabResultsPlot.tsx

```diff
@@ -32,16 +32,14 @@ const LabResultsPlot: React.FC<LabResultsPlotProps> = ({ selectedExperiments })
                         return simulationCache[cacheKey];
                     }
                     try {
-                        const simulation = await runSimulation(
-                            filteredConcs,
-                            model.parameters && Object.keys(model.parameters).length > 0
```

Comment:
Sure about removing the check here? Should we add parameters that the adapter doesn't need? (I wonder if perhaps the backend should fail if we try to run with parameters that the adapters hasn't asked for)